src/somd2/config/_config.py

```diff
@@ -867,12 +893,27 @@ def run_parallel(self, run_parallel):
             raise ValueError("'run_parallel' must be of type 'bool'")
         self._run_parallel = run_parallel
 
+    @property
+    def restart(self):
+        return self._restart
+
+    @restart.setter
+    def restart(self, restart):
+        if not isinstance(restart, bool):
+            raise ValueError("'restart' must be of type 'bool'")
+        if not restart and self.directory_existed:
+            _logger.warning(
+                f"Output directory {self.output_directory} already exists files may be overwritten"
+            )
+        self._restart = restart
+
     @property
     def output_directory(self):
         return self._output_directory
 
     @output_directory.setter
     def output_directory(self, output_directory):
+        self.cirectory_existed = False
```

Comment:
This looks like a typo, i.e. `self.directory_existed` is used in the `else` block. However, neither attribute is used anywhere in the rest of the code, so I assume they are redundant. If so, could they be removed.